### PR TITLE
Add is_visible to NodeJS and C++ Window types

### PR DESCRIPTION
**Pull request by tronical**
Amends commit 82278a574267ae2a8a85c5a076a58a9d42455734

### CODE REVIEW

**Review by ogoffart — APPROVED** [review]
There could be a test for it.  
in test/cases, a test could call a combination of show()  is_visible()  hide() in the different language.  We'd need the testing backend to report the right value.  The coverage of that test would be a bit limited since that wouldn't test the backends, just the api surface. so not sure if worth it.

**Comment by tronical** [comment]
I think you're right that it's a good idea to have some basic tests. Helps make sure the C++ API compiles. I'll add that.

File: tests/cases/examples/hello.slint [hunk 1]
```diff
@@ -170,3 +170,24 @@ Hello := Rectangle {
...
+```cpp
+auto handle = Hello::create();
+const Hello &instance = *handle;
+assert(!instance.window().is_visible());
+```
+
+
+```rust
+let instance = Hello::new();
+assert!(!instance.window().is_visible());
```
Comment (ogoffart) [inline tests/cases/examples/hello.slint:185]: i think you could call show() and hide() on the window because it is using the test backend.

**Review by ogoffart — APPROVED** [review]

File: tests/cases/examples/hello.slint (hunk 1 above)
Comment (tronical) [reply tests/cases/examples/hello.slint:189]: Good idea. Done.

**Pull request merged**